crates/pop-cli/src/commands/up/contracts_node.rs

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use clap::Args;
+use cliclack::{clear_screen, intro, set_theme};
+use duct::cmd;
+
+use crate::style::{style, Theme};
+
+const BIN_NAME: &str = "substrate-contracts-node";
+
+#[derive(Args)]
+pub(crate) struct ContractsNodeCommand;
+impl ContractsNodeCommand {
+	pub(crate) async fn execute(&self) -> anyhow::Result<()> {
+		clear_screen()?;
+		intro(format!("{}: Deploy a contracts node", style(" Pop CLI ").black().on_magenta()))?;
```

Comment:
We currently use 'deploy a contracts node,' but it doesn't sound correct. In an ongoing PR, we are already changing it to `Launch`: (https://github.com/r0gue-io/pop-cli/pull/186/files#diff-676bc9da9a6a8b135164d15340c548a94dcad29b8cd784ebda6a9e3ef2313622R37).
Could you change the message to "Launch a local contracts node"?

Reply:
Done!